### ISSUE TITLE
[Bug]: obsolete Presta / Sitemap bundle generates deprecation notices on PHP 8.1

### ISSUE THREAD

**Issue by dkarlovi** [pimcore/pimcore#11800]
### Expected behavior

No deprecations regarding Presta Sitemap Bundle.

### Actual behavior

```
Deprecated: Return type of Presta\SitemapBundle\Sitemap\XmlConstraint::count() should either be compatible with Countable::count(): int, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice
```

### Steps to reproduce

Run Pimcore 10.3.3 on PHP 8.1

This issue seems to have been fixed in SitemapBundle ^3, Pimcore requires ^2.1

**Comment by dvesh3** [comment]
@dkarlovi please see https://github.com/pimcore/pimcore/issues/11661

**Comment by brusch** [comment]
Closed in favor of #11661
